README.md

```diff
@@ -76,6 +76,15 @@ Note that lines beginning with `%` are comment lines.
 
 We also specify a number of starting types, "initial types" which can be the same or different from the base types. These follow the same format, and `atomtypes/basetypes.smarts` can be reused unless alternate behavior is desired (such as starting from more sophisticated initial types).
 
+We have two sampler options for SMARTY, one if the original and the other is elemental. The principle of sampling is the same, the only change is in which elements we are interesting to look at. Saying that, if we want to sampler all elements at the same time, we use the original sampler, if we are interested in one specific element, we can sampler an isolated element, such as Oxygen, and sampler only for that atom type.
+
+
+### Original sampler
+
+Command line example: `smarty --samplertype original --basetypes=examples/AlkEtOH/atomtypes/basetypes.smarts --initialtypes=examples/AlkEtOH/atomtypes/basetypes.smarts --decorators=examples/AlkEtOH/atomtypes/new-decorators.smarts  --molecules=examples/AlkEtOH/molecules/test_filt1_tripos.mol2 --reference=examples/AlkEtOH/molecules/test_filt1_ff.mol2 --iterations 100 --temperature=0`
+
+Original sampler is the default option. Here, smarty samples around all elements.
```

Comment:
Edit: "The original sampler is the default option. Here, smarty samples SMARTS patterns covering all elements contained in the set."